packages/clerk-js/src/ui/elements/FormControl.tsx

```diff
@@ -53,6 +64,57 @@ const getInputElementForType = (type: FormControlProps['type']) => {
   return CustomInputs[customInput] || Input;
 };
 
+function useFormTextAnimation() {
+  const prefersReducedMotion = usePrefersReducedMotion();
+
+  const getFormTextAnimation = useCallback(
+    (enterAnimation: boolean): ThemableCssProp => {
+      if (prefersReducedMotion) {
+        return {
+          animation: 'none',
+        };
+      }
+      return t => ({
+        animation: `${enterAnimation ? animations.inAnimation : animations.outAnimation} ${
+          t.transitionDuration.$slower
+        } ${t.transitionTiming.$common}`,
+        transition: `height ${t.transitionDuration.$slow}  ease`, // This is expensive but required for a smooth layout shift
```

Comment:
```suggestion
        transition: `height ${t.transitionDuration.$slow} ease`, // This is expensive but required for a smooth layout shift
```

Reply:
❓ Should easing be part of the `t.transitionDuration` interface so that it's customizable by user?